Server/CBXCUITestServer.m

```diff
@@ -83,8 +92,16 @@ + (void)start {
 - (void)start {
     NSError *error;
     BOOL serverStarted = NO;
-
-    [self.server setPort:CBX_DEFAULT_SERVER_PORT];
+    
+    NSString *portNumberString = [NSProcessInfo.processInfo.environment objectForKey: @"CbxServerPort"];
+    NSUInteger port = (NSUInteger)[portNumberString integerValue];
+    
+    if (port == 0) {
+      [self.server setPort:CBX_DEFAULT_SERVER_PORT];
```

Comment:
Do we know this will always eval to 0 when the key does not exist?

I would rather see:

```
if (!portNumberString) {

```

And this would be a good place to extract a method and write a unit test.

Reply:
fixed